resources/sdk/webmessagingjava/templates/GenesysCloudRegionHosts.mustache

```diff
@@ -0,0 +1,23 @@
+package {{invokerPackage}};
+
+public enum GenesysCloudRegionHosts {
```

Comment:
Is it feasible to genericize this and combine it with the websocket hosts file? So these hosts would only define "usw2.pure.cloud" and anything consuming the region setting would use format the URL appropriately depending on if it's being used for the API or a websocket. That would allow the consumer to specify the region once and the SDK can use it everywhere it applies.

Reply:
I've removed GenesysCloudRegionHosts and added some utility methods to GenesysCloudRegionWebSocketHosts format to an API URL.